frontend/src/scenes/annotations/AnnotationModal.tsx

```diff
@@ -110,23 +107,43 @@ export function AnnotationModal({
                                     value: AnnotationScope.Insight,
                                     label: annotationScopeToName[AnnotationScope.Insight],
                                     disabledReason:
-                                        !isInsightScoped && !onSavedInsight && 'You need to save the insight first.',
+                                        (!onSavedInsight && 'You need to save the insight first.') ||
+                                        // if existing annotation data in db (for backwards compatibility) doesn't have insight id set on it
+                                        // we can't let them change scope to insight as we don't know which insight to map to
+                                        (existingModalAnnotation
+                                            ? !existingModalAnnotation?.dashboard_item &&
+                                              "Annotation wasn't originally scoped to an insight so we can't reduce scope to insight level"
+                                            : undefined),
+                                },
+                                {
+                                    value: AnnotationScope.Dashboard,
+                                    label: annotationScopeToName[AnnotationScope.Dashboard],
+                                    tooltip: existingModalAnnotation?.dashboard_name
+                                        ? existingModalAnnotation.dashboard_name
+                                        : undefined,
+                                    disabledReason:
+                                        (!onSavedInsight && 'You need to save the insight first.') ||
+                                        // if existing annotation data in db (for backwards compatibility) doesn't have dashboard id set on it
+                                        // we can't let them change scope to dashboard as we don't know which insight to map to
+                                        (existingModalAnnotation
+                                            ? !existingModalAnnotation?.dashboard &&
+                                              "Annotation wasn't originally scoped to a dashboard so we can't reduce scope to insight level"
```

Comment:
Agree with increase/decrease. Though, just FYI, currently we don't let the user [change the dashboard of the annotation](https://github.com/PostHog/posthog/pull/23556/files#diff-c706bb2f5e7371b1bb50953b2e094138686d7a2ad56f84514d94ab6107259418R178)

This is because of the edge case:
1. User creates insight on dashboard 1 with scope = 'dashboard'.
2. Same insight is reused on dashboard 2.
3. User opens annotation on dashboard 2 and clicks Edit without changing anything -> this would then change annotation scope to dashboard 2. That's why I added the tooltip to show which dashboard scope applies to
<img width="500" alt="image" src="https://github.com/user-attachments/assets/d7b4fc32-6907-432f-9eaa-dc6fe13a2106">


Though this logic is different from that of insight where when you save, it saves scope to insight where you edited annotation. We can make this consistent and update the dashboard id as well?

Reply:
Changing to UX to show this for the edge case
<img width="579" alt="image" src="https://github.com/user-attachments/assets/a5faaba0-dbb9-4c28-a090-6e4ec604bdd9">